hoomd/md/methods.py

```diff
@@ -647,9 +716,16 @@ class NVE(_Method):
         limit (None or float): Enforce that no particle moves more than a
             distance of a limit in a single time step. Defaults to None
 
+        manifold_constraint (hoomd.md.manifold.Manifold): Manifold constraint which is
+            used by and as a trigger for the RATTLE algorithm of this method.
+            Defaults to None
+
+        tolerance (float): Defines the tolerated error particles are allowed to deviate
+            from the manifold in terms of the implicit function. Defaults to 1e-5
```

Comment:
The documentation and code do not match for tolerance in all three integration methods.

```suggestion
            from the manifold in terms of the implicit function. Defaults to 1e-6
```
Or: Change the default tolerance value to 1e-5.